WordPress/Classes/Utility/Logging/SentryStartupEvent.swift

```diff
@@ -48,9 +45,33 @@ startup time. This will block the thread. Do not use unless you're sure.
             ]
         })
 
-        client.send(event: event, completion: { _ in
+        CrashLogging.logErrorAndWait(NSError(domain: title, code: -1, userInfo: [NSLocalizedDescriptionKey: title]), userInfo: userInfo)
+    }
+}
+
+extension CrashLogging {
+    /**
+     Writes the error to the Crash Logging system, and includes a stack trace. This method will block the thread until the event is fired.
+
+     - Parameters:
+     - error: The error object
+     - userInfo: A dictionary containing additional data about this error.
+     - level: The level of severity to report in Sentry (`.error` by default)
+    */
+    static func logErrorAndWait(_ error: Error, userInfo: [String: Any]? = nil, level: SentrySeverity = .error) {
+        let event = Event(level: .error)
+        event.message = error.localizedDescription
+        event.extra = userInfo ?? (error as NSError).userInfo
```

Comment:
Since this API is only consumed by the method above ` send(title: String)`, and the NSError [instantiated here](https://github.com/wordpress-mobile/WordPress-iOS/pull/14133/files#diff-9631a66a3adecbce84ba4545aceff4e3R48) carries the same String for `domain` / `localizedDescription`, would it be possible to simplify the signature a bit?

```
static func logErrorAndWait(_ description: String, userInfo: [String: Any]? = nil, level: SentrySeverity = .error) {
```

Reply:
My idea was to kept the same signature because after this PR being merged I'll move this method to `WordPressTracks`, and in that case, it would be good to keep it generalized. Wdyt?